integration/basic_test.go

```diff
@@ -629,3 +629,48 @@ func TestDebugError(t *testing.T) {
 		require.NoError(t, db.Client().Ping(ctx, nil), "other errors should not close connection")
 	})
 }
+
+func TestPingCommand(t *testing.T) {
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t)
+	db := collection.Database()
+
+	expectedRes := bson.D{{"ok", float64(1)}}
+
+	t.Run("Multiple", func(t *testing.T) {
+		t.Parallel()
+
+		for i := 0; i < 5; i++ {
```

Comment:
What do we want to achieve there?

Reply:
That pings are not closing connection and it's possible to invoke them multiple times.